src/app/extensions/quickorder/shared/quickorder-add-products-form/quickorder-add-products-form.component.html

```diff
@@ -0,0 +1,21 @@
+<form [formGroup]="quickOrderForm" (ngSubmit)="onAddProducts()">
+  <div class="list-header d-sm-flex">
+    <div class="col-md-6 list-header-item no-seperator">{{ 'quickorder.page.product.id' | translate }}</div>
+    <div class="col-md-2 list-header-item no-seperator">{{ 'quickorder.page.product.quantity' | translate }}</div>
+  </div>
+
+  <formly-form [model]="model" [fields]="fields" [options]="options" [form]="quickOrderForm" class="pt-1"></formly-form>
+
+  <button
+    type="submit"
+    class="btn btn-primary"
+    [disabled]="quickOrderForm.pristine || quickOrderForm.invalid"
```

Comment:
@SGrueber
I removed the check for quickOrderForm.invalid, because the form validation is somehow broken from the async validator in combination of the repeating.
So this disable or enable the button in an incomprehensible way.